test/test_non_number_eval_tree_array.jl

```diff
@@ -0,0 +1,70 @@
+
+using DynamicExpressions
+using DynamicExpressions: Node, @extend_operators, OperatorEnum
+
+# SVM (scalar, vector, matrix) - struct that contains all three datatypes
+struct SVM{T}
+    dims :: Int8 # number of dimmentions
+    scalar :: T
+    vector :: Vector{T}
+    matrix :: Matrix{T}
+    SVM{T}() where {T} = new(Int8(0), zero(T), T[], T[;;])
```

Comment:
```suggestion
    SVM{T}() where {T} = new(Int8(0), zero(T), T[], zeros(T, 0, 0))
```
The `T[;;]` syntax is unavailable on earlier Julia versions (unfortunately) so to allow the tests to run on older versions, `zeros(T, 0, 0)` is needed